contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1917,7 +1917,10 @@ class tsqlBuilder : public tsqlCommonMutator
 
 		// record whether stmt is cross-db
 		if (is_cross_db)
+		{
 			stmt->is_cross_db = true;
+			is_cross_db = false;
```

Comment:
We are also setting this flag to true in `makeExecuteProcedure` but not resetting it back. What would happen if we exec a batch like
```
EXEC db.dbo.p
SELECT * FROM t
```

Reply:
can we set is_cross_db= true in enterDMLStatement ?

Reply:
Also seems like we can just remove use of this flag from `makeExecuteProcedure`